### DISCUSSION TITLE
No Results no Volume

### DISCUSSION THREAD

**Discussion by ProProgrammer-eng** [Ifsttar/I-Simpa#384] [Q&A]
Hey I need to simulate a room and it says no volume I imported it from 3ds it looks good but it has no volume and doesn't give me any results like I should need them. I have found nothing in the documentation I tried putting a box around it to define volume but it didn't help.

**Comment by mtergol** [comment]
Hello, 
have you tried to generate a meshing of the scene?
You should see the tetrahedric meshing and the correct volume in the "Properties-Information" window.

**Comment by nicolas-f** [comment]
Hi,

I don't know if there is an error message in your project, or what your room looks like. So it is really hard to guess.

You can follow the guide here:

https://i-simpa-wiki.readthedocs.io/fr/latest/import_file_recommandations.html

OpenSCAD and FreeCad gives good 3d models for the tetgen program inside I-Simpa.

**Reply by Vinom009** [reply]
I did like you said but in sketchup i didn't know how to correct it because everything is correct. Does there any mode in simpa to correct them inside the software, or is there any video how correct the problem from sketchup or in i simpa
